src/components/Reports/Reports.test.js

```diff
@@ -49,12 +49,12 @@ describe('Reports page', () => {
   it('should render domains without pages', async () => {
     const datahubClientDouble = {
       getAccountDomains: async () => fakeData,
-      getVisitsByPeriod: async () => 0,
+      getTotalVisitsOffPeriod: async () => 0,
```

Comment:
With this name `getTotalVisitsOffPeriod` you mean visits for this period? that would be with one f -->`getTotalVisitsOfPeriod`

Reply:
Else you are saying visits outside a period.

Reply:
thanks very much. i updated now